cluster/manifests/z-karpenter/deployment.yaml

```diff
@@ -51,7 +51,8 @@ spec:
                 - ALL
             readOnlyRootFilesystem: true
           {{if eq .Cluster.ConfigItems.karpenter_version "current"}}
-          image: "container-registry.zalando.net/teapot/karpenter:0.37.0-main-26.patched"
+          # patched version with ipv6adresscount support
+          image: "container-registry-test.zalando.net/teapot/karpenter:0.37.0-pr-26-3.patched"
```

Comment:
Using a PR image here is not desired. We need to integrate this change in the prod image for this to be mergable.

Reply:
fixed